modules/dnn/src/ocl4dnn/include/default_kernel_config.hpp

```diff
@@ -0,0 +1,290 @@
+#ifndef _OPENCV_OCL4DNN_DEFAULT_KERNEL_CONFIG_HPP_
+#define _OPENCV_OCL4DNN_DEFAULT_KERNEL_CONFIG_HPP_
+#define CONFIG_NUM (sizeof(default_kernel_config) / sizeof(char *) / 2)
+const char *default_kernel_config[] = {
+// Configs for 72 EUs device
+"631d9d8eea647618","6 3 16 2 1 1 16 1 0 ",
```

Comment:
Could you expand these hashes? (avoid using of hashes here)

Something like this (probably without driver version until we support version masks):
```
Intel(R) Corporation: Intel(R) HD Graphics (VER=r5.0.63503) CU=24 k5x1_cn32_g1_s1x1_d1x1_b0_in64x64_p2x0_num1_M32
```

How did you handle different names for the same devices on different platforms?
For example, we have this for i7-6700K with Intel OpenCL runtime:
- [Windows](http://pullrequest.opencv.org/buildbot/builders/precommit_opencl/builds/12523/steps/test_dnn-ippicv-opencl/logs/stdio): "Intel(R) HD Graphics 530"
- [Linux](): "Intel(R) HD Graphics"

Are there configurations for "beignet" runtime?

Could you also add OpenCL information for mentioned devices (into comments) from OpenCV tests header?

Reply:
-- Could you expand these hashes?
Yes, I will use more readable string to replace hashes.

-- How did you handle different names for the same devices on different platform?
Good question. I will remove the device name and vendor name from the the signature.

--Are there configurations for "beignet" runtime?
No. We recommend to use  "Intel OpenCL Driver" (https://software.intel.com/en-us/articles/opencl-drivers#latest_linux_driver) for the performance consideration. Beignet lacks of some key extension supports that need by convolution kernel.

--Could you also add OpenCL information for mentioned devices (into comments) from OpenCV tests header?
Ok.